addons/hr_holidays/static/src/owl/components/thread/thread.js

```diff
@@ -0,0 +1,51 @@
+odoo.define('hr_holidays.component.Thread', function (require) {
+'use strict';
+
+var Thread = require('mail.component.Thread');
+
+var core = require('web.core');
+const utils = require('web.utils');
+
+var _t = core._t;
+
+utils.patch(Thread, 'hr_holidays_thread', {
+    /**
+     * @override
+     */
+    _useStore: function (state, props) {
+        const res = this._super(...arguments);
+        const directPartnerLocalId = res.thread
+            ? res.thread.directPartnerLocalId
+            : undefined;
+        const directPartner = directPartnerLocalId
+            ? state.partners[directPartnerLocalId]
+            : undefined;
+        return Object.assign({}, res, {
+            directPartner,
+        });
+    },
+    /**
+     * Returns the "out of office" text for the direct partner of the thread if
+     * applicable.
+     *
+     * @returns {string}
+     */
+    getOutOfOfficeText: function () {
+        if (!this.storeProps.directPartner) {
+            return undefined;
+        }
+        if (!this.storeProps.directPartner.out_of_office_date_end) {
+            return undefined;
+        }
```

Comment:
would return empty string, to be consistent with docstring

Reply:
ok (anyway not a case that will happen considering where it is used, but just to be safe)